config/core/build-configs.yaml

```diff
@@ -251,6 +251,29 @@ fragments:
       - 'CONFIG_MMC_SDHCI=y'
       - 'CONFIG_MMC_SDHCI_PCI=y'
       - 'CONFIG_MMC_SDHCI_ACPI=y'
+      - 'CONFIG_CHROME_PLATFORMS=y'
+      - 'CONFIG_CHROMEOS_LAPTOP=y'
+      - 'CONFIG_CHROMEOS_TBMC=y'
+      - 'CONFIG_DM_CRYPT=y'
+      - 'CONFIG_DM_THIN_PROVISIONING=y'
+      - 'CONFIG_DM_INIT=y'
+      - 'CONFIG_DM_VERITY=y'
+      - 'CONFIG_CFS_BANDWIDTH=y'
+      - 'CONFIG_USER_NS=y'
+      - 'CONFIG_BLK_DEV_NVME=y'
+      - 'CONFIG_CROS_EC=y'
+      - 'CONFIG_CROS_EC_LPC=y'
+      - 'CONFIG_CROS_EC_I2C=y'
+      - 'CONFIG_IIO=m'
+      - 'CONFIG_IIO_CROS_EC_SENSORS_CORE=m'
+      - 'CONFIG_IIO_CROS_EC_SENSORS=m'
+      - 'CONFIG_MFD_CROS_EC_DEV=m'
+      - 'CONFIG_TCG_TPM=y'
+      - 'CONFIG_TCG_TIS=y'
+      - 'CONFIG_TCG_TIS_SPI=y'
+      - 'CONFIG_TCG_TIS_SPI_CR50=y'
+      - 'CONFIG_TCG_TIS_I2C_CR50=y'
```

Comment:
These might be required to enable some tpm tests, but right now, AFAIK we don't have a running test suite for this.

Reply:
@eballetbo Do we need a separate test suite just for TPM? I'm adding some boot sanity checks to bootrr which I think should be enough for now to test these configs, eg check that dev node exists, check TPM chip communication, check TCG version etc.

Reply:
OK so let's drop them for now and consider adding them if/when we actually add a TPM test suite.

Reply:
Great, I'll drop these and add them via a separate review once we merge some TPM tests.